### ISSUE TITLE
The reports on GEAR are now not accessible via screen reader

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#257]
In a previous accessibility review meeting with Dan on 1/24/24, he indicated that the [IT Standards report page](https://dev1.ea.gsa.gov/#/it_standards) was accessible. His screen reader told him that he landed in a table with 8 columns/11 rows, and he said he was able to navigate it successfully. Through our efforts in addressing the accessibility since then, we may have regressed that accessibility of all of the report views on GEAR. This ticket is to make the tables navigable via screen reader again.
